### PR TITLE
docs cron: update meta files unconditionally

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Comment by garyverhaegen-da** [comment]
Obviously the comment is garbage; I just reverted #8385.

File: ci/cron/src/Main.hs [hunk 1]
```diff
@@ -463,4 +463,9 @@ main = do
       Docs -> do
           docs sdkDocOpts
           docs damlOnSqlDocOpts
+          -- FIXME: this is meant to run once to create the `latest` file
```
Comment (cocreature) [inline ci/cron/src/Main.hs:466]: the comment sounds wrong, we want to update `snapshots.json`.

Comment (cocreature) [reply ci/cron/src/Main.hs:466]: Also not sure if at this point it isn’t just better to keep this code long term.

**Review by cocreature — APPROVED** [review]
Thanks!

File: ci/cron/src/Main.hs (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/cron/src/Main.hs:466]: I'll merge as is to fix the issue tonight; I'll think a bit more about what to do longer-term.

Comment (cocreature) [reply ci/cron/src/Main.hs:466]: :+1:

**Comment by garyverhaegen-da** [comment]
This does not seem to have worked 🤔

**Comment by garyverhaegen-da** [comment]
Caching was getting in the way, the situation is now resolved.

**Pull request merged**